src/libServer/LookupServer.cpp

```diff
@@ -1369,13 +1363,10 @@ Json::Value LookupServer::GetShardingStructure() {
 
     unsigned int num_shards = shards.size();
 
-    if (num_shards == 0) {
-      throw JsonRpcException(RPC_IN_WARMUP, "No shards yet");
-    } else {
-      for (unsigned int i = 0; i < num_shards; i++) {
-        _json["NumPeers"].append(static_cast<unsigned int>(shards[i].size()));
-      }
+    for (unsigned int i = 0; i < num_shards; i++) {
```

Comment:
As discussed, it is better to keep the check here.

Reply:
Please check the previous comment.